dev-docs/show-prebid-ads-on-amp-pages.md

```diff
@@ -125,7 +125,38 @@ This is the creative that your Ad Ops team needs to upload to the ad server (it'
 {: .alert.alert-success :}
 You can always get the latest version of the creative code below from [the AMP example creative file in our GitHub repo](https://github.com/prebid/prebid-universal-creative/blob/master/template/amp/dfp-creative.html).
 
-{% include dev-docs/amp-creative.md %}
+{: .alert.alert-danger :}
+Replace **MACRO** in the code below with the appropriate macro for the ad server you're using. For example, for DFP replace **MACRO** with **PATTERN**.
+
+{% highlight html %}
+
+    <script src="https://cdn.jsdelivr.net/npm/prebid-universal-creative@latest/dist/creative.js"></script>
+    <script>
+      var ucTagData = {};
+      ucTagData.adServerDomain = "";
+      ucTagData.pubUrl = "%%MACRO:url%%";
+
+      // if DFP, use this param
```

Comment:
Should we be calling this GAM now? Maybe GAM (formerly DFP)?

Reply:
I'd prefer to hold off on this now and update from DFP --> GAM in one fell swoop.